pyproject.toml

```diff
@@ -9,8 +9,8 @@ author-email = "thomas@kluyver.me.uk"
 home-page = "https://github.com/takluyver/terminado"
 description-file = "README.rst"
 requires = [
-    "ptyprocess;os_name==posix",
-    "pywinpty;os_name==nt,python_version>='3.0'",
+    "ptyprocess;os_name=='posix'",
+    "pywinpty;os_name=='nt',python_version>='3.5'",
```

Comment:
Thanks. I'm just looking through the spec, and I think the way to combine expressions is with `and`, rather than comma. I've opened an issue on flit to validate these markers at build time. https://github.com/takluyver/flit/issues/152